### DISCUSSION TITLE
Egress Volume

### DISCUSSION THREAD

**Discussion by jonathanlane** [nhost/nhost#2238] [Q&A]
Hi,

Our project is under development and has very low data and database volumes at the moment. There are just 2 developers working on a daily basis and no other users.

We have upgraded to Pro subscription which allows up to 50Gb of egress, but I see that we've used nearly 20Gb of that already. However, when I look at the network traffic in Grafana, it shows only 200Mb of outbound network traffic for the month. I've no idea why the egress is so high.

Does anyone have any tips or suggestions to help us find out what we've done to create so much egress volume?

Thanks

**Comment by dbarrosop** [comment]
If you share the subdomain (feel free to DM me on discord) we can take a look but egress traffic consists of three parts:

1. CDN traffic
2. Egress traffic from the cloud (i.e. storage files if there is a MISS in the CDN, hsaura responses, auth responses, etc)
3. Database traffic if using the connection string (i.e. doing a pgdump from your machine or connecting some application directly to it)

So when counting egress make sure you look at the three parts.
